src/routes/index.js

```diff
@@ -7,6 +7,8 @@ const router = express.Router();
 const members = require('./member');
 // const superAdmins = require('./super-admins');
 // const classes = require('./class');
+// const trainers = require('./trainer');
+// const activities = require('./activity');
 const trainers = require('./trainer');
```

Comment:
Here in the comment lines 10 and 11 it's the same at 12 y 13, maybe delete the comment ones.

Reply:
Thanks!.. Those commented lines were deleted.